Subject: libkeymap: fix double kbdfile open

976867c3a84 (Rewrite libkbdfile library) changed kbdfile_find to error
when the file descriptor was already opened. Make sure kbdfile_find
isn't called again when kbdfile is found in include_dirpath1.

--- a/src/libkeymap/analyze.l
+++ b/src/libkeymap/analyze.l
@@ -177,7 +177,7 @@ find_standard_incl_file(struct lk_ctx *ctx, char *s, struct kbdfile *fp)
 	i--;
 	pathname = kbdfile_get_pathname(ctx->stack[i]);
 
-	if (kbdfile_find(s, include_dirpath1, include_suffixes, fp)) {
+	if (rc = kbdfile_find(s, include_dirpath1, include_suffixes, fp)) {
 		if ((rc = find_incl_file_near_fn(ctx, s, pathname, fp)) == -1)
 			return rc;
 	}
